### PR TITLE
Add mbs-frontend image

### PR DESCRIPTION
**Pull request by Zlopez**
Implements part of the acceptance criteria for #48

Signed-off-by: Michal Konečný <mkonecny@redhat.com>

### CODE REVIEW

File: images/mbs-frontend/mbs.wsgi [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import logging
+logging.basicConfig(level=logging.DEBUG)
```
Comment (odra) [inline images/mbs-frontend/mbs.wsgi:2]: I will file an issue to take the log level value from an env var but we can merge as it is

**Comment by odra** [comment]
entrypoint.sh is not an executable, it needs perm changes:

```sh
$ docker run -it quay.io/fedora/mbs-frontend:latest
/bin/sh: /entrypoint.sh: Permission denied
```

File: images/mbs-frontend/mbs.wsgi (hunk 1 above)
Comment (Zlopez) [reply images/mbs-frontend/mbs.wsgi:2]: I just copied what was in the `mbs.wsgi`.

**Comment by Zlopez** [comment]
I made it executable, so it should work now

**Pull request merged**